src/utils.cpp

```diff
@@ -1102,6 +1102,11 @@ int utils::run_interactively(
 	} else if (status == 127) {
 		LOG(level::DEBUG, "%s: couldn't run shell", caller);
 	}
+	if (status != 0) {
```

Comment:
Imagine if `status` is 127. We'd get two log messages: one specific one ("couldn't run shell") and one generic one ("cmd X returned Y"). The second one isn't helpful. The same if `status` is -1.

So this `if` can be turned into an `else` branch of the previous `if`, so that generic log message is only shown if we don't have a specific one.